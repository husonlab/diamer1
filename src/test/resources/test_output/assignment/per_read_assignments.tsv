6
ReadID	OVO (0.20) kmer count	OVO (0.20) norm. kmer count	OVO (0.50) kmer count	OVO (0.50) norm. kmer count	OVO (0.60) kmer count	OVO (0.60) norm. kmer count	OVO (0.80) kmer count	OVO (0.80) norm. kmer count	OVO (0.90) kmer count	OVO (0.90) norm. kmer count	OVO (1.00) kmer count	OVO (1.00) norm. kmer count	OVA (0.70) kmer count	OVA (0.70) norm. kmer count
0	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)	(1 from root) rank2 (2)
1														
2														
3	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)
4	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)	(2 from root) organism3 (5)
5	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)	(no rank) root (0)
