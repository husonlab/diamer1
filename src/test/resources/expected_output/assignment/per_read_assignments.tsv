6
ReadID	OVO (ratio: 0.200000)	OVO (ratio: 0.800000)
0	(1 from root) rank2 (2)	(1 from root) rank2 (2)
1		
2		
3	(1 from root) rank1 (1)	(1 from root) rank1 (1)
4	(2 from root) organism3 (5)	(2 from root) organism3 (5)
5	(no rank) root (0)	(no rank) root (0)
